### PR TITLE
FRR BGP NBI support bug fixes & authentication password encryption support for neighbor & peer-group

### PR DESCRIPTION
**Pull request by venkatmahalingam**
Summary:
1)	Fix for no-more not working in show command
2)	BGP neighbor/peer-group authentication password with encrypted option
3)	CLI client json.loads fix to improve json parsing performance
4)	Few display nbr fixes and route-map leaf-list count 1 check when it is associated with nbr/peer-group.

### CODE REVIEW

File: models/yang/extensions/openconfig-bgp-ext.yang [hunk 1]
```diff
@@ -553,6 +554,47 @@ module openconfig-bgp-ext {
...
+              "Authentication password for the neighbor group, 
+              this will be changed to encrypted text internally 
+              when the encrypted flag is configured with false.";
+      }
+
+      leaf encrypted {
+          type boolean;
+          default "false";
+          description
+              "This leaf indicates whether the auth-password is encrypted text,
```
Comment (prvattem) [inline models/yang/extensions/openconfig-bgp-ext.yang:570]: Change auth-password to password.

File: models/yang/extensions/openconfig-bgp-ext.yang [hunk 2]
```diff
@@ -553,6 +554,47 @@ module openconfig-bgp-ext {
...
+  grouping bgp-ext-neighbor-group-auth-password {
+      description
+          "Configuration parameters related to authentication password for neighbor group.";
+
+      container auth-password {
+          description
+              "Parameters relating to BGP authentication password";
+          container config {
+              description
+                  "Configuration information for BGP neighbor group authentication routes.";
```
Comment (prvattem) [inline models/yang/extensions/openconfig-bgp-ext.yang:585]: Please remove the word "routes"

File: models/yang/extensions/openconfig-bgp-ext.yang (hunk 1 above)
Comment (venkatmahalingam) [reply models/yang/extensions/openconfig-bgp-ext.yang:570]: ok

File: models/yang/extensions/openconfig-bgp-ext.yang (hunk 2 above)
Comment (venkatmahalingam) [reply models/yang/extensions/openconfig-bgp-ext.yang:585]: ok

**Review by joyas-joseph — APPROVED** [review]

**Comment by joyas-joseph** [comment]
Comments will be addressed in next commit.

**Pull request merged**